StackMin.py

```diff
@@ -0,0 +1,33 @@
+class Stack:
+    def __init__(self):
+        self.dataStack = []  
+        self.minStack = []   
```

Comment:
Use snake case instead of camel case in Python.
```suggestion
        self.data_stack = []  
        self.min_stack = []   
```